style.css

```diff
@@ -440,11 +471,11 @@ filter: invert(15%);
 
 .container, .tips{
 	position: relative;
-	width: 275px;
-	height: 275px;
+	width: 314px;
+	height: 314px;
 	display: inline-block;
 	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-	margin: 0px 5px 10px 5px;
+	margin: 0px -4px -4px -4px;

```

Comment:
I have removed the gaps between the images to see how it would look, i believe it can be improved